xarray/backends/zarr.py

```diff
@@ -1558,7 +1560,7 @@ def open_zarr(
     if from_array_kwargs is None:
         from_array_kwargs = {}
 
-    if chunks == "auto":
+    if chunks is _default:
```

Comment:
do we need to raise a deprecation warning for `chunks = "auto"`? That does change behavior when passed explicitly, so may be worth having the deprecation cycle.

Edit: actually, from https://github.com/pydata/xarray/pull/11010#discussion_r2620829165 it looks like we may not need the deprecation cycle